--- outek1	2013-10-18 12:16:30.000000000 +0200
+++ outek2	2013-10-18 12:17:38.000000000 +0200
@@ -3,7 +3,7 @@ Array
     [tid] => 136
     [vid] => 14
     [name] => Agd kuchnia
-    [description] => 
+    [description] => opis angielski
     [format] => filtered_html
     [weight] => 0
     [vocabulary_machine_name] => epesi_category
@@ -39,9 +39,9 @@ Array
                 (
                     [0] => Array
                         (
-                            [value] => AGD
+                            [value] => Agd kuchnia
                             [format] => 
-                            [safe_value] => AGD
+                            [safe_value] => Agd kuchnia
                         )
 
                 )
@@ -54,10 +54,11 @@ Array
                 (
                     [0] => Array
                         (
-                            [value] => 
+                            [value] => opis angielski
                             [summary] => 
                             [format] => filtered_html
-                            [safe_value] => 
+                            [safe_value] => <p>opis angielski</p>
+
                             [safe_summary] => 
                         )
 
@@ -67,13 +68,12 @@ Array
                 (
                     [0] => Array
                         (
-                            [value] => test3
-                            [summary] => test4
+                            [value] => opis polski
+                            [summary] => 
                             [format] => filtered_html
-                            [safe_value] => <p>test3</p>
-
-                            [safe_summary] => <p>test4</p>
+                            [safe_value] => <p>opis polski</p>
 
+                            [safe_summary] => 
                         )
 
                 )
@@ -83,14 +83,40 @@ Array
     [name_original] => Agd kuchnia
     [translations] => Array
         (
-            [original] => 
+            [original] => en
             [data] => Array
                 (
+                    [en] => Array
+                        (
+                            [entity_type] => taxonomy_term
+                            [entity_id] => 136
+                            [language] => en
+                            [source] => 
+                            [uid] => 1
+                            [status] => 1
+                            [translate] => 0
+                            [created] => 1382091389
+                            [changed] => 1382091448
+                        )
+
+                    [pl] => Array
+                        (
+                            [entity_type] => taxonomy_term
+                            [entity_id] => 136
+                            [language] => pl
+                            [source] => en
+                            [uid] => 1
+                            [status] => 1
+                            [translate] => 0
+                            [created] => 1382091430
+                            [changed] => 1382091430
+                        )
+
                 )
 
         )
 
-    [description_original] => dupa
+    [description_original] => opis angielski
 )
 Array
 (
